layers/+tools/command-log/packages.el

```diff
@@ -14,7 +14,7 @@
       ;; is an open PR to merge the fork into the original repo.
       ;; TODO when the PR is merged upstream, change to use the original package
       ;; from MELPA (IOW remove :location argument)
-      '((command-log-mode :location (recipe :fetcher github
+      '((command-log-mode :location (command-log-mode :fetcher github
```

Comment:
I didn't dig into the github fetcher part, but quelpa use the package build library from melpa to deal with the github fetcher. Since melpa uses the same recipe format as I mentioned, I think this should be the correct format of the command-log-mode recipe